src/replication_log.h

```diff
@@ -0,0 +1,51 @@
+#ifndef	REPLICATION_LOG_H
+#define	REPLICATION_LOG_H
+
+enum replication_log_level {
+	LOG_LEVEL_ERR,
+	LOG_LEVEL_INFO,
+	LOG_LEVEL_DEBUG,
+	LOG_LEVEL_MAX,
+};
+
+int replication_log_level;
+
+#define REPLICA_LOG(fmt, ...)						\
+{									\
+	if (replication_log_level <= LOG_LEVEL_INFO)			\
```

Comment:
these conditions should be reverse if I'm not wrong

Reply:
yes